examples/network_profile/isolation_with_external_subnet/README.md

```diff
@@ -0,0 +1,37 @@
+# network_profile example
+
+This is an example on how to create a Network Profile with isolation using external subnet.
+
+## Getting Started
+
+There are variables which need to be added to terraform.tfvars. The first are for connecting to the vRealize Automation (vRA) endpoint. The others are names of cloud_account, and region that already setup in vRA.
+
+* `url` - The URL for the vRealize Automation (vRA) endpoint
+* `refresh_token` - The refresh token for the vRA account
+* `cloud_account` - The name of the cloud account added in vRA
+* `region` - The region within the cloud account
+* `subnet-name` - The subnet to add to Network Profile
```

Comment:
This is across all of these examples, we should standardize using underscores for all variables and not use dashes. I found this interesting best practice site: https://www.terraform-best-practices.com/naming

Reply:
Thanks for sharing the best practice site. I'm working on replacing all the dashes with underscores from these examples. I will update the PR soon after I'm done.